### PR TITLE
Maintainers Article from Steve Smith via Hai Ah Nam

### PR DESCRIPTION
**Pull request by bernhold**
Extracted from #481
Came from BSSw Blog Planning board: https://github.com/bernhold/bssw-blog-planning/issues/6

### CODE REVIEW

**Comment by hnamLANL** [comment]
sorry I didn't follow the editorial process :(
Will do so now.  Hopefully no more changes.

File: Articles/Blog/2020-03-maintainers-drive-sustainability.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+ParFlow has been in continuous use for scientific discovery since 1992, and several of the current ParFlow developers have been involved with the project since the first equation was written on a white board.  The ParFlow project has undergone long-term development and has been a fertile ground to gain a deep appreciation of the need for solid software engineering practices to improve scientific productivity. 
+
+**Limited software development practices.**  The experiences of ParFlow’s development team matches an [earlier BSSw blog on continuous technology refreshment](https://bssw.io/blog_posts/continuous-technology-refreshment-an-introduction-using-recent-tech-refresh-experiences-on-visit) (CTR), which highlights a core issue - applications that expect to continue to work over time cannot be static, but must undergo ‘*periodic upgrade or replacement of infrastructure to deliver continued reliability, improved speed, capacity, and/or new features*' to respond to the developing needs of users and workflows.  Prior to this project, the ParFlow team had not adopted formal software engineering practices including CTR due to the lack of funding targeted at software maintenance.
```
Comment (bernhold) [inline Articles/Blog/2020-03-maintainers-drive-sustainability.md:30]: Text says "Prior to this project..."

Which project is that?  IDEAS-Classic?  That's first mentioned much later.  Need change something here.

Comment (hnamLANL) [reply Articles/Blog/2020-03-maintainers-drive-sustainability.md:30]: Yes - IDEAS-Classic.
Let's revise to remove "Prior to this project" and start with "The ParFlow team..."

Comment (bernhold) [reply Articles/Blog/2020-03-maintainers-drive-sustainability.md]: Done.  Thanks

File: Articles/Blog/2020-03-maintainers-drive-sustainability.md [hunk 2]
```diff
@@ -8,60 +8,60 @@
...
 
-**Intermittent funding for development and maintenance of the software.** A key lesson that we have made out of observing the development and evolution of ParFlow has been the need to have one or more identified maintainers to keep the software in a running and usable state.   Looking back at the on/off cycle of funding cycle we have observed several recurring patterns of failure when there are no maintainers.   
+**Intermittent funding for development and maintenance of the software.** A key lesson that we have learned from observing the development and evolution of ParFlow has been the need to have one or more identified maintainers to keep the software in a running and usable state.   Looking back at the on/off cycle of funding cycle, we have observed several recurring patterns of failure when there are no maintainers.
```
Comment (prwolfe) [inline Articles/Blog/2020-03-maintainers-drive-sustainability.md:34]: "cycle of funding cycle" should be cycle of funding.

Comment (prwolfe) [reply Articles/Blog/2020-03-maintainers-drive-sustainability.md:34]: or just "funding cycle"

File: Articles/Blog/2020-03-maintainers-drive-sustainability.md [hunk 3]
```diff
@@ -8,60 +8,60 @@
...
       
-All of these practices and tools have made the role of maintainer easier, more distributed, more automated, and more sustainable with limited budgets.  Will ParFlow be around in another 25 years?  We have no idea but then no-one is more surprised at ParFlow’s longevity than the original developers.   You never know where a project might take you when you start.
+All of these practices and tools have made the role of maintainer easier, more distributed, more automated, and more sustainable with limited budgets.  Will ParFlow be around in another 25 years?  We have no idea, but then no-one is more surprised at ParFlow’s longevity than the original developers.   You never know where a project might take you when you start.
 
 **More about ParFlow:**
```
Comment (prwolfe) [inline Articles/Blog/2020-03-maintainers-drive-sustainability.md:67]: Do we have a link or other info for this?

**Comment by prwolfe** [comment]
David,

Your question about content rather than editing - I think this matches my current day-to-day efforts on CTH after a long period of neglect.  I think it fits well.

Paul

File: Articles/Blog/2020-03-maintainers-drive-sustainability.md (hunk 3 above)
Comment (bernhold) [reply Articles/Blog/2020-03-maintainers-drive-sustainability.md:67]: @prwolfe there are links to the parflow web site and GH repo in the table below.  We can make them clickable.  Does that address your question?  Or are you looking for something else?

Comment (prwolfe) [reply Articles/Blog/2020-03-maintainers-drive-sustainability.md:67]: Looks much better - Thanks!

File: Articles/Blog/2020-03-maintainers-drive-sustainability.md (hunk 2 above)
Comment (bernhold) [reply Articles/Blog/2020-03-maintainers-drive-sustainability.md]: Done

**Pull request merged**